lib/rules/aria-hidden-focus-matches.js

```diff
@@ -0,0 +1,13 @@
+const { getComposedParent } = axe.commons.dom;
+
+function shouldMatchElement(el) {
```

Comment:
We should probably document the purpose of this matcher. Something like "Only catch the outer-most aria-hidden=true element."